README.md

```diff
@@ -55,6 +55,25 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
+For example, it's possible to insert a link to the online CodeCogs equation editor;
+below includes syntax to be read on darkmode by default in D-Wave's [Leap IDE](https://ide.dwavesys.io).
+
+`<img style="filter:invert(1)" src="https://latex.codecogs.com/svg.latex?\large\,x_{ik}+x_{jk}-2x_{ik}x_{jk}"> `
+
+The above renders the following expression in your github markdown readme.
+
+<img style="filter:invert(1)" src="https://latex.codecogs.com/svg.latex?\large\,x_{ik}+x_{jk}-2x_{ik}x_{jk}"> 
+
+Inserting a relative link to the svg in the repository in the `readme_imgs`
+folder produces the same result, but without the filter option.
```

Comment:
Perhaps instead of stating "but without the filter option", we could suggest using a white background here.